File: src/main/resources/output/IFC4X3_java/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelAggregates/IfcRelAggregates.java
public class IfcRelAggregates extends IfcRelDecomposes {
    public IfcObjectDefinition RelatingObject;
    public SET [1:?] OF IfcObjectDefinition RelatedObjects;

    // === WHERE CLAUSES ===
    // NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelAggregates/IfcRelAggregates.js
class IfcRelAggregates extends IfcRelDecomposes {
    constructor() {
        /** @type {IFCOBJECTDEFINITION} */
        this.RelatingObject = null;
        /** @type {SET [1:?] OF IFCOBJECTDEFINITION} */
        this.RelatedObjects = null;
    }

    // === WHERE CLAUSES ===
    // NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
}
